### PR TITLE
feat(metrics): status tag for session duration

### PR DESCRIPTION
**Pull request by ig-ra**

### CODE REVIEW

File: internal/backend/sessions/sessionworkflows/workflows.go [hunk 1]
```diff
@@ -217,9 +217,15 @@ func (ws *workflows) sessionWorkflow(wctx workflow.Context, params *sessionWorkf
...
+
 		prints, err = runWorkflow(wctx, l, ws, data, params.Debug)
+
 		duration = time.Since(startTime)
-		sessionDurationHistogram.Record(ctx, duration.Milliseconds())
+		sessionStatus := "ok"
+		if err != nil {
+			sessionStatus = "error"
+		}
+		sessionDurationHistogram.Record(ctx, duration.Milliseconds(), telemetry.WithLabels("status", sessionStatus))
```
Comment (itayd) [inline internal/backend/sessions/sessionworkflows/workflows.go:228]: also add replay

Comment (efiShtain) [reply internal/backend/sessions/sessionworkflows/workflows.go:228]: why replay?
we count the total session run from created to final state
replay is just an internal implementation detail

Comment (ig-ra) [reply internal/backend/sessions/sessionworkflows/workflows.go:228]: @itayd 
moved to https://linear.app/autokitteh/issue/ENG-1462/metrics-3. Resolving here

File: internal/backend/sessions/sessionworkflows/workflows.go [hunk 2]
```diff
@@ -217,9 +217,15 @@ func (ws *workflows) sessionWorkflow(wctx workflow.Context, params *sessionWorkf
 
 		startTime = data.Session.CreatedAt()
 		l.Info("session workflow: started", fields...)
+
 		prints, err = runWorkflow(wctx, l, ws, data, params.Debug)
+
 		duration = time.Since(startTime)
-		sessionDurationHistogram.Record(ctx, duration.Milliseconds())
+		sessionStatus := "ok"
```
Comment (efiShtain) [inline internal/backend/sessions/sessionworkflows/workflows.go]: succeed / failed

**Review by efiShtain — APPROVED** [review]

**Pull request merged**